enhancements/kubelet/virtualization-higher-workload-density.md

```diff
@@ -0,0 +1,424 @@
+---
+title: virtualization-higher-workload-density
+authors:
+  - enp0s3
+  - fabiand
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - TBD
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Virtualization higher workload density
+
+## Summary
+
+Fit more workloads onto a give node - achieve a higher workload density - by overcommitting it's memory resources. Due to timeline needs a two-phased approach is considered.
+
+## Motivation
+
+Today, OpenShift Virtualization is reserving memory (`requests.memory`) according to the needs of the virtual machine itself. However, usually an application within the virtual machine does not utilize _all_ the memory _all_ the time. Instead, only _sometimes_ there are memory spikes within the virtual machine. Extrapolating this behavior to all virtual machines on a given node leads to the observation that _most often_ there is no memory pressure and a low memory utilization.
+Underutilized hardware resources are a cost factor to end-users.
+
+This proposal is about increasing the workload - virtual machine - density per node, in order to increase the memory utilization, in order to reduce the cost per virtual machine.
+
+> [!NOTE]
+> CPU resources are already over-committed in OpenShift today, thus the limiting factor - usually - are memory resources.
+
+### User Stories
+
+* As an administrator, I want to be able to enable higher workload density for OpenShift Virtualization clusters so that I can increase the memory utilization.
```

Comment:
My point would be: The risk of instability arises usually from over-committment.
Without overcommittment we should be good.

Leaving it out, because: SWAP can help for those situations, but this EP is about higher density.